api/pom.xml

```diff
@@ -64,6 +64,29 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
+                <configuration>
+                        <failOnError>false</failOnError>
+                        <additionalparam>${javadoc.options}</additionalparam>
+                        <docfilessubdirs>true</docfilessubdirs>
+                        <javadocDirectory>${project.basedir}/../src/main/javadoc</javadocDirectory>
+                        <attach>true</attach>
+                        <doclint>none</doclint>
+                        <doctitle>${project.name}</doctitle>
+                        <windowtitle>${project.name}</windowtitle>
+                        <header><![CDATA[<br>${project.name} v${project.version}]]></header>
+                        <bottom>
+<![CDATA[
```

Comment:
We can replace this with:

```xml
<bottom>
    <![CDATA[
        Copyright &#169; {inceptionYear}&#x2013;{currentYear} {organizationName}. All rights reserved.
        <br>
        Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">license terms</a>.
    ]]>
</bottom>
```

And set the `<inceptionYear>2017</inceptionYear>` in the Parent POM. The other properties are retrieved from the `organization` section.

Reply:
Good suggestions! As for the inceptionYear, since most of the spec asciidoc for the specs started 2016. I have set the inceptionYear to 2016.

Reply:
Great :)